app/models/user_stat.rb

```diff
@@ -215,7 +215,7 @@ def self.update_draft_count(user_id = nil)
       SQL
 
       MessageBus.publish(
-        '/user',
+        "/user-drafts/#{user_id}",
```

Comment:
Will this change affect current implementations?

Reply:
Unlikely, as far as I can tell this was added for drafts only when I look up the commit history. Also the specs are passing so this gives the confidence that no plugins are relying on this.